server/__tests__/users.controler.test.ts

```diff
@@ -0,0 +1,52 @@
+import { connectDB, clearDB, closeDB } from "./testdb";
+import request from "supertest";
+import app from "../src/utils/app";
+import { createTestUser } from "./factories/users.factory";
+import { HTTP_STATUS } from "../src/utils/httpStatus";
+import { tryPromise } from "../src/utils/inlineHandlers";
+import { getSessionByUserId } from "../src/models/session.model";
+
+
+describe.only("RENAME ME", () => {
+    beforeAll(async () => {
+		await connectDB();
+	});
+
+	beforeEach(async () => {
+		await clearDB()
+	})
+
+	afterAll(async () => {
+		await clearDB();
+		await closeDB();
+	});
+    
+    it("I NEED A NEW NAME", async () => {
+        const { user, sessionToken } = await logInUserAndGetSessionToken();
+
+        const res = await request(app)
+            .put(`/users/${user._id}`)
+            .set("Cookie", [`sessionToken=${sessionToken.data?._id}`])
```

Comment:
Kinda need this in order to have your auth flow work in tests. Kinda annoying as you can't call set before setting the path and method :(